MuxExoPlayer/src/main/java/com/mux/stats/sdk/muxstats/MuxBaseExoPlayer.java

```diff
@@ -285,6 +294,13 @@ public void setStreamType(int type) {
   @Override
   public void dispatch(IEvent event) {
     if (player != null && player.get() != null && muxStats != null) {
+      numberOfEventsSent ++;
```

Comment:
Can you remove the whitespace between the variable name and the `++`? This should be handled by the code formatter.